packetbeat/sniffer/afpacket_linux.go

```diff
@@ -40,6 +43,91 @@ type afpacketHandle struct {
 	promiscPreviousStateDetected bool
 	device                       string
 	log                          *logp.Logger
+	metrics                      *metrics
+}
+
+type metrics struct {
+	unregisterFn func()
```

Comment:
Conventionally, these are simply `unregister`. Same non-Hungarian for `done` below.